scss/_navbar.scss

```diff
@@ -160,7 +158,8 @@
           flex-direction: row;
 
           .dropdown-menu {
-            position: absolute;
+            position: absolute !important;
```

Comment:
Yep, I know.
But now, because of mobile first abordation, we need to overwrite [L80](https://github.com/zalog/bootstrap/blob/bfbb443b9e1bd305afdc17b8c557606e64639f03/scss/_navbar.scss#L80).

Why mobile first abordation:
- for navbars that never collapse, `.navbar.navbar-expand`. My first navbar in the _before_ example
- for navbars that always collapse, just `.navbar` class. My third navbar example in that _before_ like